packages/shared/src/lib/auth.ts

```diff
@@ -8,6 +8,21 @@ import {
   KratosMethod,
 } from './kratos';
 
+export enum EventNames {
```

Comment:
As this is an exported enum, I think it can be good to make it more explicit as some other areas might have their own `EventNames` as well.
```suggestion
export enum AuthEventNames {
```

Reply:
yeah actually thought about that.
since it always comes from "auth" file I thought it would clean up, but maybe we should be more explicit indeed for future proofing.